nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/service/AbstractAuditBackend.java

```diff
@@ -460,4 +470,31 @@ protected Predicate convert(FilterMapEntry entry) {
         return queryLogs(builder);
     }
 
+    @Override
+    public void append(List<String> jsonEntries) {
+        ObjectMapper mapper = new ObjectMapper();
+        List<LogEntry> logEntries = jsonEntries.stream().map(json -> {
+            try {
+                return mapper.readValue(json, LogEntryImpl.class);
+            } catch (IOException e) {
+                throw new NuxeoException("Invalid json logEntry: " + json, e);
+            }
+        }).collect(Collectors.toList());
+        addLogEntries(logEntries);
+    }
+
+    @Override
+    public ScrollResult scroll(AuditQueryBuilder queryBuilder, int batchSize, int keepAlive) {
+
+        cursorService.checkForTimedOutScroll();
+        List<LogEntry> logEntries = queryLogs(queryBuilder);
+        String scrollId = cursorService.registerCursor(logEntries.iterator(), batchSize, keepAlive);
```

Comment:
Here you're gonna load probably a lot of entries in memory, the scroll will only be on the local `List`. Does it what we want ?

Reply:
We can use it as a default implementation, but the next step is to have each backend overriding it to use its own scrolling API if possible (e.g Elasticsearch).

Reply:
👍